src/include/meta.h

```diff
@@ -87,7 +87,6 @@
 /*
  * Block based incremental backup structure. These live in the connection.
  */
-#define WT_BLKINCR_MAX 2
```

Comment:
Is it expected to move this one? Is it defined at the right spot? I can see it is used across multiple modules.